airflow/www/views.py

```diff
@@ -3737,7 +3709,6 @@ def datasets_summary(self):
             )
 
     @expose("/robots.txt")
-    @action_logging
```

Comment:
We need to leave this one - it powers [this warning](https://github.com/apache/airflow/blob/d5d6f4b1885a99f5a5e3063dbd54e1f32700c1df/airflow/www/views.py#L1020-L1029).

Reply:
FIxed